src/DotNet.Status.Web/Controllers/TelemetryController.cs

```diff
@@ -0,0 +1,67 @@
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using Microsoft.DotNet.Kusto;
+using Kusto.Ingest;
+using System.Collections.Generic;
+using System;
+
+namespace DotNet.Status.Web.Controllers
+{
+    [Route("api/telemetry")]
+    [ApiController]
+    public class TelemetryController : ControllerBase
+    {
+        private readonly ILogger<TelemetryController> _logger;
+        private readonly IOptionsSnapshot<KustoOptions> _options;
+        private readonly IKustoIngestClient _client;
+
+        public TelemetryController(
+            ILogger<TelemetryController> logger,
+            IOptionsSnapshot<KustoOptions> options,
+            IKustoIngestClient client)
+        {
+            _logger = logger;
+            _options = options;
+            _client = client;
+        }
+
+        [HttpPost("collect/arcade-validation")]
+        public async Task<IActionResult> CollectArcadeValidation([Required] ArcadeValidationData data)
+        {
+            KustoOptions options = _options.Value;
+
+            if (string.IsNullOrEmpty(options.KustoIngestConnectionString))
+            {
+                throw new Exception("No KustoIngestConnectionString set");
```

Comment:
I'd probably call this an "InvalidOperationException".  A base level "Exception" can be hard to track (since you can't catch it without catching literally all other exceptions, which is probably not intended).

Reply:
Yeah, that's much better.